shared/dev/dumb-sheet.render.desktop.js

```diff
@@ -43,24 +43,29 @@ class Render extends Component<void, any, any> {
           />
         </Box>
         {Object.keys(dumbComponentMap).map(key => {
-          if (this.state.filter && key.toLowerCase().indexOf(this.state.filter) === -1) {
+          const map = dumbComponentMap[key]
```

Comment:
this makes the filter smarter. you can pass 'tracker' to get all the tracker views. you can pass 'nonuser' to get just those subitems